Formula/root.rb

```diff
@@ -52,10 +48,19 @@ class Root < Formula
   depends_on "xz" # for LZMA
   depends_on "zstd"
 
+  uses_from_macos "libxml2"
+
   conflicts_with "glew", because: "root ships its own copy of glew"
 
   skip_clean "bin"
 
+  # Can be removed post 6.22.08
+  patch :p0 do
+    url "https://patch-diff.githubusercontent.com/raw/root-project/root/pull/7494.patch"
```

Comment:
Let's use the patch at

https://github.com/root-project/root/commit/d113c9fcf7e1d88c573717c676aa4b97f1db2ea2.patch?full_index=1

This uses the full commit hash, so the URL is guaranteed to be stable. (Well, barring a GitHub catastrophe.)

Reply:
I didn't realize you could use a commit directly as a patch rather than a PR, sounds good. Should we see if this builds first, or just change it (depends on if the old builds get cancelled correctly, I expect?)

Reply:
The build wasn't started, since the tap_syntax test didn't like the current patch URL. You can change it now to get a build going.

Reply:
Oh, nice, the tap_sytnax cancelled things. Pushing.